openedx/features/colaraz_features/views.py

```diff
@@ -156,6 +183,23 @@ class CourseAccessRoleUpdateView(BaseViewMixin, CourseAccessRoleFilterMixin, Col
 
     success_url = reverse_lazy('colaraz_features:course-access-roles-list')
 
+    def get_object(self):
+        queryset = super(CourseAccessRoleUpdateView, self).get_queryset()
+
+        pk = self.kwargs.get(self.pk_url_kwarg, '')
+        try:
+            ids = [int(key) for key in pk.split(',') if key.strip()]
+        except (TypeError, ValueError):
+            raise HttpBadRequest('Invalid Request URL')
+        else:
+            if not ids:
+                raise HttpBadRequest('Invalid Request URL')
+
```

Comment:
```suggestion

            if not ids:
                raise HttpBadRequest('Invalid Request URL')

```
this else block seems unnecessary.

Reply:
It will be helpful when ids will be an empty list.

Reply:
I am just talking about the `else:` keyword. I mean we can have that `if not ids` check without the else.

Reply:
updated.